sources/api/netdog/Cargo.toml

```diff
@@ -1,13 +1,18 @@
 [package]
 name = "netdog"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
We've historically not done much management of subcomponent versioning. I'm supportive of using meaningful versioning for these changes, especially for subcomponents that will get their own RPM.